.github/workflows/release.yml

```diff
@@ -0,0 +1,173 @@
+            echo "Adding aarch64-unknown-linux-gnu target"
+            sudo apt-get update -y
+            sudo apt-get install -y make g++ libssl-dev gcc-aarch64-linux-gnu
+            rustup target add aarch64-unknown-linux-gnu
+          else
+            echo "Compiling for ${{matrix.target}}"
+          fi
+
+      - name: Install Rust (Windows)
+        if: ${{ startsWith(matrix.os, 'windows') }}
+        run: |
+          curl -sSf -o rustup-init.exe https://win.rustup.rs
+          ./rustup-init.exe -y --default-toolchain ${{ matrix.rustup_toolchain }} --default-host ${{ matrix.target }}
+          echo "##vso[task.setvariable variable=PATH;]%PATH%;%USERPROFILE%\.cargo\bin"
+
+      - name: Cargo build
+        run: cargo build --release --target ${{ matrix.target }}
+
+      - name: Archive (UNIX)
+        run: |
+          mkdir -p artifacts
+          cp -av target/${{ matrix.target }}/release/zola .
+          tar -czf ${{ github.event.repository.name }}-${{ github.ref_name }}-${{ matrix.target }}.tar.gz zola
+        if: ${{ ! startsWith(matrix.os, 'windows') }}
+
+      - name: Archive (Windows)
+        run: |
+          mkdir -p artifacts
+          cp target/${{ matrix.target }}/release/zola.exe .
+          tar -czf ${{ github.event.repository.name }}-${{ github.ref_name }}-${{ matrix.target }}.zip zola.exe
+        if: ${{ startsWith(matrix.os, 'windows') }}
+
+      - name: Attest Build Provenance
+        uses: actions/attest-build-provenance@v1
+        continue-on-error: true
+        with:
+          subject-path: ${{ github.event.repository.name }}-${{ github.ref_name }}-${{ matrix.target }}.*
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: ${{ github.event.repository.name }}-${{ github.ref_name }}-${{ matrix.target }}
+          path: ${{ github.event.repository.name }}-${{ github.ref_name }}-${{ matrix.target }}.*
+          if-no-files-found: error
+          retention-days: 7
+
+  Release:
+    needs: [Release-Build]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - name: Ensure artifacts dir exists
+        run: mkdir -p artifacts
+
+      - name: Download Artifact
+        uses: actions/download-artifact@v4
+        with:
+          path: artifacts
+          merge-multiple: true
+
+      - name: Release
+        uses: softprops/action-gh-release@c062e08bd532815e2082a85e87e3ef29c3e6d191
+        with:
+          name: ${{ github.ref_name }}
+          tag_name: ${{ github.ref_name }}
+          generate_release_notes: true
+          fail_on_unmatched_files: true
+          body: |
+            Welcome to this new release of Zola ${{ github.ref_name }}!
+
+            All artifacts are signed with this repos identity using Sigstore.
+            You can verify the signatures using the `GitHub` CLI.
+
+            ```shell
+            gh attestation verify --owner ${{ github.repository_owner }} <my-artifact>
```

Comment:
Right now Azure generates a list of commits since the last release (https://github.com/getzola/zola/releases/tag/v0.19.2), is it possible to keep it?

Reply:
Yeah it should generate release notes with `generate_release_notes: true`. My demo release didn't have any real changes since it was the first release and I'm not working with PRs for active development and debugging.

I'm also using this action here: https://github.com/hegerdes/joplin-plugin-remote-note-pull/releases/tag/v1.3.0 and Pulumi also uses uses with some more changes than my small project, see https://github.com/pulumi/pulumi-awsx/releases

I think they might look a litte different to azures notes though.

Feel fee to add whatever you want to that release body you are the owner, it was just an example for now.